terraform-scripts/quarkus-roq.tf

```diff
@@ -1,36 +1,36 @@
 # Create repository
-resource "github_repository" "quarkus_statiq" {
-  name                   = "quarkus-statiq"
+resource "github_repository" "quarkus_roq" {
```

Comment:
Renaming the resource key will trigger the resource destruction. Let me rename the state in Terraform and re-run this PR

Reply:
Sure

Reply:
Actually, keep the `quarkus_statiq` resource key instead, I can deal with the state management later :)